cli/docs/cli.json

```
@@ -1,6 +1,7 @@
 {
   "name": "oxide",
   "version": "0.5.0+20240502.0",
+  "about": "Control an Oxide environment",
```

Optional nit, "control" sounds a bit weird? Maybe it's just me

```suggestion
  "about": "Manage an Oxide environment",
```